Wikipedia/Code/AppearanceSettingsViewController.swift

```diff
@@ -0,0 +1,183 @@
+import UIKit
+
+@objc public protocol WMFAppearanceSettingsViewControllerDelegate {
+    func themeChangedInController(_ controller: AppearanceSettingsViewController, theme: Theme)
```

Comment:
This should be removed, the `WMFUserDidSelectThemeNotification` should cause `WMFAppViewController` to properly update the theme - there could be a bug in the app controllers code if something isn't getting properly updated.